### ISSUE TITLE
Unexpected error on initial integration setup.

### ISSUE THREAD

**Issue by aberg83** [jason0x43/hacs-hubitat#8]
I get the following error message in the HA logs after trying to setup the integration the first time.

Unexpected exception
Traceback (most recent call last):
  File "/config/custom_components/hubitat/config_flow.py", line 70, in async_step_user
    info = await validate_input(user_input)
  File "/config/custom_components/hubitat/config_flow.py", line 48, in validate_input
    "id": hub.id,
AttributeError: 'Hub' object has no attribute 'id'

**Comment by rhd** [comment]
@aberg83 I remember seeing some errors in my log before I updated HA to the 0.105.2... Which version of HA are you on?

**Comment by jason0x43** [comment]
Actually, that looks like a bug. It would only affect new installs as it occurs during initial setup. The integration is trying access a property that no longer exists on a hub object.

I'll have a fix out at some point today.

**Comment by jason0x43** [comment]
Resolved in v0.3.1 (4d2864e)

**Comment by SPCulhane** [comment]
I'm experiencing a similar issue:

```
2020-02-13 14:05:17 ERROR (MainThread) [custom_components.hubitat.config_flow] Unexpected exception
Traceback (most recent call last):
  File "/config/custom_components/hubitat/config_flow.py", line 51, in async_step_user
    info = await validate_input(self.hass, user_input)
  File "/config/custom_components/hubitat/config_flow.py", line 35, in validate_input
    "id": hub.id,
AttributeError: 'Hub' object has no attribute 'id'
```

HA: 0.105.3
HassOS: 3.10
hacs-hubitat - v0.3.1
